pkg/storage/sasNode.go

```diff
@@ -24,16 +24,19 @@ import (
 	"os"
 	"os/exec"
 	"strings"
+	"time"
 
 	saslib "github.com/Seagate/csi-lib-sas/sas"
 	"github.com/Seagate/seagate-exos-x-csi/pkg/common"
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	"github.com/pkg/errors"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
 )
 
+var globalRemovedDevicesMap = map[string]time.Time{}
```

Comment:
Suggest a comment indicating this map is indexed by Volume WWN and stores a timestamp when the volume is published.